test/leSpec.js

```diff
@@ -411,10 +525,24 @@ describe('print option', function () {
         /*jshint -W020 */
         XDomainRequest = XMLHttpRequest; //trick into thinking we are in IE8/9 browser
         /*jshint +W020 */
-        LE.log('some message');
+        IOPS.log('some message');
         expect(console.log.mostRecentCall.args[0]).toMatch(/[0-9a-z]{8} some message/);
     });
 
     afterEach(restoreXMLHttpRequests);
     afterEach(destroy);
 });
+
+function fail(message) {
+    throw message;
+}
+
+function assert(condition, message) {
```

Comment:
We're the using the [Jasmine](https://jasmine.github.io/) test framework in this project, so you can use its assertions (`expect(actual).toBe(expected)`).
It will give you nice messages when tests fail, and report those failures properly in CI builds (jenkins or whatever).
For custom messages, you can do `expect(actual).toBe(expected, 'my custom message')`